hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/recordcache/RecordCacheImpl.java

```diff
@@ -353,7 +353,7 @@ public List<TransactionRecord> getRecords(@NonNull final AccountID accountID) {
     private WritableStates getWritableState() {
```

Comment:
if the previous suggestion is applied, as workingStateAccessor does not have a MerkleState anymore:
```
    private WritableStates getWritableState() {
        final var state = workingStateAccessor.getMerkleState();
        if (state == null) {
            throw new RuntimeException("State is null. This can only happen very early during bootstrapping");
        }
        return state.getWritableStates(NAME);
    }
```

Reply:
Done